apps/website/src/components/home/features/compose/composeFeature.tsx

```diff
@@ -8,9 +8,9 @@ export const ComposeFeature = () => {
   const [codeVisible, setCodeVisible] = useState(false);
   return (
     <FeatureSection
-      feature={compose}
```

Comment:
Did this ever work @cfraz89 ? feature wasn't a field on the object.

Reply:
Oh you can delete that whole compose folder, its legacy code now